js/Features.js

```diff
@@ -52,19 +42,19 @@ var Features = function (config) {
 
     self._features = features;
     self._featureMask = 0;
-}
+};
```

Comment:
No, leave it then. Since we all have access to WebStorm now, using the default style proposed by WebStorm as the 'official' style is a good approach. I hadn't realised that some of the changes were based on WebStorm when I wrote this comment.